lib/DefaultResolver.php

```diff
@@ -214,6 +218,8 @@ private function doResolve($name, array $types, $options) {
             $this->config = (yield \Amp\resolve($this->loadResolvConf()));
         }
 
+        $config = $this->loadEnvConfig();
```

Comment:
Yes, it may. But we shouldn't allow runtime change of the options that way. It's not the way to do that. It should always be `/etc/resolv.con` + original `RES_OPTIONS`